notification/app/notification/services/WindowsNotificationSender.scala

```diff
@@ -30,9 +33,21 @@ class WindowsNotificationSender(hubClient: NotificationHubClient, configuration:
     }
   }
 
-
-  private def getCounts(destination: Either[Topic, UserId]): Future[RepositoryResult[Int]] = destination match {
-    case Left(topic: Topic) => topicSubscriptionsRepository.count(topic)
+  private def count(destination: Destination): Future[RepositoryResult[Int]] = destination match {
+    case Left(topics: Set[Topic]) => sumOf(topics)
     case Right(_: UserId) => Future.successful(RepositoryResult(1))
   }
+
+  private def sumOf(topics: Set[Topic]): Future[RepositoryResult[Int]] = {
```

Comment:
It's either the sum of everything or it returns an error, so sumOf seems relevant.